eng/common/scripts/modules/Package-Properties.psm1

```diff
@@ -107,7 +107,16 @@ function Extract-JsPkgProps ($pkgPath, $serviceName, $pkgName)
     if (Test-Path $projectPath)
     {
         $projectJson = Get-Content $projectPath | ConvertFrom-Json
-        $jsStylePkgName = $pkgName.replace("azure-", "@azure/")
+        $jsStylePkgName = ""
```

Comment:
I don't believe this code is used any more. You should be able to update https://github.com/Azure/azure-sdk-for-js/blob/master/eng/scripts/Language-Settings.ps1.